employee-rostering-frontend/.env

```diff
@@ -1 +1 @@
-REACT_APP_BACKEND_URL=.
+REACT_APP_BACKEND_URL=
```

Comment:
Was the `.` the reason why Cypress test failed?

Reply:
Yep; when set to '.', it uses the realitive url, which due to this PR, becomes something like '/{tenantId}/skills/rest/....'.

When set to ''. it uses the base, so it becomes '/rest/...'. It worked when I tested on standalone jar with this change.

Reply:
(also ran the cypress test locally, and it fixes the test)